### DISCUSSION TITLE
Fooocus support?

### DISCUSSION THREAD

**Discussion by juh9870** [AbdBarho/stable-diffusion-webui-docker#627] [Ideas]
It would be spectacular to have a support for the [Fooocus](https://github.com/lllyasviel/Fooocus) UI

**Comment by digitallamb** [comment]
FYI, I just created a PR to add support for Fooocus

**Reply by mashb1t** [reply]
nice, thanks! I can also support here, now one the maintainers of Fooocus.
See also https://github.com/lllyasviel/Fooocus/pull/1418

Link to PR: https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/635
